### PR TITLE
Add mobile view for drawer

### PR DESCRIPTION
**Pull request by JasonChong96**
## Problem

Currently, the drawer's fields on mobile are really small due to the amount of text. 

## Solution

Update the design of the drawer to match the temp fix for both desktop and mobile
Removed access borders on mobile user table

## Before & After Screenshots

**AFTER**:

[Mobile](https://user-images.githubusercontent.com/35889982/82654713-be786000-9c53-11ea-9478-0330f07dcac9.png)
[Desktop](https://user-images.githubusercontent.com/35889982/82655001-2169f700-9c54-11ea-90fd-f1bd21cf82de.png)

### CODE REVIEW

File: src/client/components/UserPage/Drawer/ControlPanel/index.tsx [hunk 1]
```diff
@@ -31,27 +36,64 @@ const useStyles = makeStyles(() =>
     dialogContents: {
       marginTop: 116,
       marginBottom: 141,
+      [theme.breakpoints.down('sm')]: {
+        marginTop: theme.spacing(6.5),
+      },
```
Comment (kylerwsm) [inline src/client/components/UserPage/Drawer/ControlPanel/index.tsx:41]: Can we flip the breakpoints around to only use upwards breakpoints to maintain our mobile-first approach?

File: src/client/components/UserPage/Drawer/ControlPanel/widgets/ConfigOption.tsx [hunk 2]
```diff
@@ -33,28 +37,44 @@ export enum TrailingPosition {
...
 }
 
 // Represents an edit option on the ControlPanel.
 export default function ConfigOption(props: ConfigOptionProps) {
   const classes = useStyles({ trailingPosition: props.trailingPosition })
   return (
-    <main className={classes.mainContainer}>
+    <main
+      className={classes.mainContainer}
+      style={{ flexWrap: props.wrapTrailing ? 'wrap' : 'nowrap' }}
```
Comment (kylerwsm) [inline src/client/components/UserPage/Drawer/ControlPanel/widgets/ConfigOption.tsx:56]: Can we put inline stylings inside their respective classes?

File: src/client/components/UserPage/Drawer/ControlPanel/DrawerHeader.tsx [hunk 3]
```diff
@@ -26,10 +33,12 @@ const useStyles = makeStyles(() =>
 export default function DrawerHeader() {
   const classes = useStyles()
```
Comment (kylerwsm) [inline src/client/components/UserPage/Drawer/ControlPanel/DrawerHeader.tsx:34]: For mobile view, I believe there needs to be more padding between the close icon and the copy link button.

**Comment by JasonChong96** [comment]
Addressed changes

**Pull request merged**